README.md

```diff
@@ -1,2 +1,19 @@
 # microverse-hello-world-project
 My first project for Hello World day 2 activity.
+This project is part of an activity that helps me to get aquainted with the GitHub flow, Linters and pull requests.
+
+Built With
+HTML & CSS
+
+Authors
+GitHub: @githubhandle
+Twitter: @twitterhandle
+LinkedIn: LinkedIn
+
+Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
```

Comment:
Your readme file should be descriptive as stated in the project requirement 👍 this makes your repository look professional 💯